src/main/java/github/weichware10/analyse/gui/TrialSelector.java

```diff
@@ -0,0 +1,129 @@
+package github.weichware10.analyse.gui;
+
+import github.weichware10.analyse.Main;
+import github.weichware10.analyse.gui.util.AbsScene;
+import github.weichware10.util.Logger;
+import github.weichware10.util.ToolType;
+import github.weichware10.util.data.TrialData;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.List;
+import java.util.Optional;
+
+import javax.crypto.SecretKey;
+
+import javafx.application.Platform;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Button;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Dialog;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'javafx.scene.control.Dialog' import. Should be before 'javax.crypto.SecretKey'.